ct.yaml

```diff
@@ -8,6 +8,7 @@ chart-repos:
   - grafana=https://grafana.github.io/helm-charts
   - prometheus-community=https://prometheus-community.github.io/helm-charts
   - stable=https://charts.helm.sh/stable
+  - kube-state-metrics=https://kubernetes.github.io/kube-state-metrics
```

Comment:
Are bitnami charts still required anywhere else in the repo? If not it could be removed from here.